src/compiler/CodeGen.cpp

```diff
@@ -47,22 +48,14 @@ namespace Theta {
 
         // TODO: This wont work if we have nested operations on either side
         if (binOpNode->getLeft()->getNodeType() == ASTNode::Types::STRING_LITERAL) {
-            return BinaryenStringConcat(
-                module,
-                binaryenLeft,
-                binaryenRight
-            );
+            return BinaryenStringConcat(module, binaryenLeft, binaryenRight);
```

Comment:
Is there any way to make this reformatting? Often times when I write each function argument on its own line its to improve readability. In the case above it would make sense for it to be reformatted to this. However [here](https://github.com/alexdovzhanyn/ThetaLang/pull/10/files#diff-9fb0c005b36ccb73533df22b3d2ce07d5c80ec02f80880cc6de0673fb78bc101R70) I would have wanted to keep it on separate lines.

Reply:
there is an option for this : https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignafteropenbracket but i didn't get it to work even with the "Align" value. Though i managed to solve the mentioned problem by reducing the maxColumnLimit so that it forces it to destruct it's paramatres into separate lines.(only for the long example). Maybe needs more looking into. I'm still learning clang-format so there could be an option that i'm missing